src/main/java/org/zone/region/flag/player/entitydamage/EntityDamagePlayerFlag.java

```diff
@@ -0,0 +1,54 @@
+package org.zone.region.flag.player.entitydamage;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.zone.region.flag.Flag;
+import org.zone.region.flag.FlagType;
+import org.zone.region.flag.FlagTypes;
+import org.zone.region.group.key.GroupKey;
+import org.zone.region.group.key.GroupKeys;
+
+import java.util.Optional;
+
+public class EntityDamagePlayerFlag implements Flag.TaggedFlag, Flag.AffectsPlayer, Flag.Enabled {
```

Comment:
A tag flag cannot hold data and therefore should not implement enabled too

Reply:
k